wgpu-core/src/hub.rs

```diff
@@ -271,16 +271,19 @@ impl<I: TypedId + Debug> IdentityHandler<I> for Mutex<IdentityManager> {
 
 pub trait IdentityHandlerFactory<I> {
     type Filter: IdentityHandler<I>;
-    fn spawn(&self) -> Self::Filter;
+    fn spawn(&self, min_index: Index) -> Self::Filter;
 }
 
 #[derive(Debug)]
 pub struct IdentityManagerFactory;
 
 impl<I: TypedId + Debug> IdentityHandlerFactory<I> for IdentityManagerFactory {
     type Filter = Mutex<IdentityManager>;
-    fn spawn(&self) -> Self::Filter {
-        Mutex::new(IdentityManager::default())
+    fn spawn(&self, min_index: Index) -> Self::Filter {
```

Comment:
Could you explain why `min_index` is useful for `Backend::Empty` only? It seems like we're trying to avoid colliding with `SurfaceId::ERROR` but I'm wondering why it's not necessary to do this for other IDs and backends

Reply:
I should put a comment in there. The reason is because `Empty == 0`. So only ID=0 with Epoch=0 on Empty backend ends up with u64 value == 0. Other backends are guaranteed to be non-zero in raw values.